modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-hwevents_{context}"]
+= Configuring bare-metal events monitoring using PolicyGenTemplate CRs
+
+You can configure bare-metal hardware events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Create a Git repository where you manage your custom site configuration data.
+
+[NOTE]
+====
+Multiple `HardwareEvent` resources are not permitted.
+====
+
+.Procedure
+
+. To configure the AMQ Interconnect Operator and the Bare Metal Event Relay Operator, add the following YAML to `spec.sourceFiles` in the `common-ranGen.yaml` file:
++
+[source,yaml]
+----
+# AMQ interconnect operator for fast events
+- fileName: AmqSubscriptionNS.yaml
+  policyName: "amq-sub-policy"
+- fileName: AmqSubscriptionOperGroup.yaml
+  policyName: "amq-sub-policy"
+- fileName: AmqSubscription.yaml
+  policyName: "amq-sub-policy"
+# Bare Metal Event Rely operator
+- fileName: BareMetalEventRelaySubscriptionNS.yaml
+  policyName: "bare-metal-events-sub-policy"
+- fileName: BareMetalEventRelaySubscriptionOperGroup.yaml
+  policyName: "bare-metal-events-sub-policy"
+- fileName: BareMetalEventRelaySubscription.yaml
+  policyName: "bare-metal-events-sub-policy"
+----
+
+. Add the `AmqInstance` and `HardwareEvent` CRs to the `spec.sourceFiles` field in your specific site YAML file, for example, in the `example-sno-site.yaml` file:
++
+[source,yaml]
+----
+- fileName: AmqInstance.yaml
+  policyName: "config-amq-policy"
+- fileName: HardwareEvent.yaml
+  policyName: "config-bare-metal-events-policy"
+  spec:
+    nodeSelector: {}
+    transportHost: "amqp://amq-router.amq-router.svc.cluster.local" <1>
+    logLevel: "debug"
+----
+<1> Ensure that the `transportHost` field in the `HardwareEvent` CR matches the `name` and `namespace` of AMQ Interconnect.
```

Comment:
- namespace is not specified on this line.
- can we document how to find out the expected name and namespace?
- Also in ZTP, the AMQ Interconnect CR is created under amq-router namespace by default, while HarewareEvent CR is created under openshift-bare-metal-events namespace by default. @jzding could you please confirm the expected configs to document for these two CRs?

Reply:
Perhaps there is a better way to describe this.
The transportHost here is set to:
```
transportHost: "amqp://<name of AMQ Interconnect CR>.<AMQ Interconnect namespace>.svc.cluster.local"
```
Since by default:
`name of AMQ Interconnect CR` is set to `amq-router`
`AMQ Interconnect namespace` is set to `amq-router`

We have:
```
transportHost: "amqp://amq-router.amq-router.svc.cluster.local"
```

Reply:
Updated it as `transportHost: "amqp://<amq_interconnect_name>.<amq_interconnect_namespace>.svc.cluster.local"` to fit our doc guidelines as well.

Reply:
Could you please add a description for this field? 
e.g., transportHost url is composed using existing AMQ Interconnect CR name and namespace, such as transportHost: "amqp://amq-router.amq-router.svc.cluster.local" when AMQ Interconnect name and namespace are both set to "amq-router".